package.json

```diff
@@ -32,6 +32,7 @@
     "mochawesome-merge": "^4.2.0",
     "mochawesome-report-generator": "^6.2.0",
     "node-sass": "^6.0.1",
+    "sass": "^1.59.3",
```

Comment:
In the future - Do not add extra files to PR
To add the file to a commit use git add src/.